tensorflow_addons/layers/normalizations_test.py

```diff
@@ -157,31 +178,19 @@ def _create_and_fit_Sequential_model(self, layer, shape):
         model.fit(x=input_batch, y=output_batch, epochs=1, batch_size=1)
         return model
 
-    def test_weights(self):
-        # Check if weights get initialized correctly
-        layer = GroupNormalization(groups=1, scale=False, center=False)
-        layer.build((None, 3, 4))
-        self.assertEqual(len(layer.trainable_weights), 0)
-        self.assertEqual(len(layer.weights), 0)
-
-        layer = InstanceNormalization()
-        layer.build((None, 3, 4))
-        self.assertEqual(len(layer.trainable_weights), 2)
-        self.assertEqual(len(layer.weights), 2)
-
-    def test_apply_normalization(self):
-        input_shape = (1, 4)
-        reshaped_inputs = tf.constant([[[2.0, 2.0], [3.0, 3.0]]])
-        layer = GroupNormalization(groups=2, axis=1, scale=False, center=False)
-        normalized_input = layer._apply_normalization(reshaped_inputs, input_shape)
-        self.assertTrue(
-            np.all(
-                np.equal(
-                    self.evaluate(normalized_input),
-                    np.array([[[0.0, 0.0], [0.0, 0.0]]]),
-                )
-            )
-        )
+    # def test_apply_normalization(self):
+    #     input_shape = (1, 4)
+    #     reshaped_inputs = tf.constant([[[2.0, 2.0], [3.0, 3.0]]])
+    #     layer = GroupNormalization(groups=2, axis=1, scale=False, center=False)
+    #     normalized_input = layer._apply_normalization(reshaped_inputs, input_shape)
+    #     self.assertTrue(
+    #         np.all(
+    #             np.equal(
+    #                 self.evaluate(normalized_input),
+    #                 np.array([[[0.0, 0.0], [0.0, 0.0]]]),
+    #             )
+    #         )
+    #     )
```

Comment:
Maybe you forgot to remove this?

Reply:
Ops, my bad. Thanks @gabrieldemarmiesse.